app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Helpers\TaskStatus;
+use App\Models\Task;
+use App\Models\User;
+use App\Rules\ValidTaskType;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\File;
+
+class TaskController extends Controller
+{
+    /**
+     * Show the application task dashboard.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index($activeFilter = null)
+    {
+
+        $this->authorize('update', Task::class);
+
+        $user = auth()->user();
```

Comment:
suggestion: fetch the user through the service provider instead, and specify correct return type.
```suggestion
    /**
     * Show the application task dashboard.
     */
    public function index(Authenticatable $user, string $activeFilter = null): View
    {
        $this->authorize('update', Task::class);
```